tutorials/cyber/raw_data_generator/test/test_raw_data_generator.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import os
+from raw_data_generator import raw_data_generator
+import sys
+from mockito import mock, when
+
+class RawDataGeneratorTest(unittest.TestCase):
+
+    def setUp(self):
+        sys.modules['HDFileSystem'] = mock()
+
+    def test_data_generator(self):
+        cwd = os.getcwd()
+        output_dir = "/test/data/output"
```

Comment:
should the output directory for test data go to `/data/lanl/output` instead of `/test`? It would help keep things cleaner if users work with persistent volumes.

Reply:
Sure I will move it there